node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/InfraDeployerImpl.java

```diff
@@ -88,7 +88,7 @@ public void prepare() {
                 Version targetVersion = infrastructureVersions.getTargetVersionFor(nodeType);
                 hostSpecs = provisioner.prepare(
                         application.getApplicationId(),
-                        application.getClusterSpecWithVersion(targetVersion),
+                        application.getClusterSpecWithVersion(targetVersion, Optional.empty()), // TODO: Use dockerImageRepo from zk
```

Comment:
Does this apply here? We are potentially allocating new nodes here, which do not have any docker image associated with them. Unless you mean the docker image from https://github.com/vespa-engine/vespa/blob/master/node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/DockerImages.java
?

Reply:
No, probably not. I'll just remove it for now.